pkg/products/observability/reconciler.go

```diff
@@ -152,6 +153,22 @@ func (r *Reconciler) reconcileSecrets(_ context.Context, _ k8sclient.Client, _ *
 }
 
 func (r *Reconciler) reconcileComponents(ctx context.Context, serverClient k8sclient.Client, _ *integreatlyv1alpha1.RHMI, productNamespace string) (integreatlyv1alpha1.StatusPhase, error) {
+	// Create a ConfigMap in the operator namespace to prevent observability CR from being created in the operator ns.
+	cfgMap := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "observability-operator-no-init",
+			Namespace: r.Config.GetOperatorNamespace(),
+		},
+	}
+	op, err := controllerutil.CreateOrUpdate(ctx, serverClient, cfgMap, func() error {
+		return nil
+	})
+	if err != nil {
+		return integreatlyv1alpha1.PhaseInProgress, err
```

Comment:
@carlkyrillos if there is an error in the creation of the config map let's return an PhaseFailure

Reply:
Good catch, I changed it to PhaseFailed